ocamltest/run_unix.c

```diff
@@ -258,7 +258,7 @@ static int handle_process_termination(
 #endif /* WCOREDUMP */
   corestr = core ? "" : "no ";
   fprintf(stderr,
-    "Process %d got signal %d(%s), %score dumped\n",
+    "Process %ld got signal %d(%s), %score dumped\n",
```

Comment:
This isn't a portable fix - neither `%d` nor `%ld` is generally correct for printing a `pid_t`. I think the right thing is to use `%ld`, but replace the `pid` argument with `(long)pid`.